bundles/org.openhab.ui/web/src/components/widgets/system/oh-icon.vue

```diff
@@ -56,6 +69,7 @@ export default {
     iconName () {
       const icon = (this.context) ? this.config.icon : this.icon
       if (!(typeof icon === 'string' || icon instanceof String)) return ''
+      if (icon.indexOf('oh:classic:') === 0) return icon.substring(icon.indexOf('oh:classic:') + 11)
```

Comment:
Any OH icon set has to be supported, not only classic. Is it the case?

Reply:
No. How can any OH iconset be retrieved from the server? Under which name?

Reply:
By providing the set name in the "iconset" request parameter. The request URL must not contain the icon set. 
So for icon toto in set titi, the request should look like icon/toto&iconset=titi.
Or icon/temperature?iconset=classic for oh:classic:temperature.

Reply:
If iconset parameter is not provided, the classic icon set is considered by default by the OH server,